packages/workspace/src/browser/quick-open-workspace.ts

```diff
@@ -80,7 +82,7 @@ export class QuickOpenWorkspace implements QuickOpenModel {
         this.items = [];
         this.workspaceService.recentWorkspaces().then(workspaceRoots => {
             if (workspaceRoots) {
```

Comment:
this is not part of your change, but `workspaceRoots` is an array and wouldn't be null/undefined as per the return type of `workspaceService.recentWorkspaces`. therefore there's no need to do the if-check.